test/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 include(GNUInstallDirs)
+if (CMAKE_GENERATOR MATCHES "Visual Studio")
+set(OQS_PROV_BINARY_DIR ${CMAKE_BINARY_DIR}/lib/Debug)
```

Comment:
Is this path always correct -- particularly if CMAKE_BUILD_TYPE is set to "Release"? Or does that variable not propagate into VS?

Reply:
I set the cmake build type to Debug in ci, which seems a reasonable choice to produce more output. So yes

Reply:
What happens when someone sets the build type to something different?

Reply:
Improved in https://github.com/open-quantum-safe/oqs-provider/pull/263/commits/ce60ecd5e0640f834cb0e41a0049eeb6cc9dcbee